api/app/signals/apps/reporting/csv/utils.py

```diff
@@ -39,6 +40,23 @@ def zip_csv_files(files_to_zip: list, using: str) -> None:
                 )
 
 
+def rotate_zip_files(using: str, max_csv_amount: int = 30) -> None:
+    """
+    rotate csv zip file in the {now:%Y}/{now:%m}/{now:%d} folder
+
+    :returns:
+    """
```

Comment:
Do we need a feature flag to turn this behaviour on and off?

Reply:
Don't think so, because this is only enabled by setting a celery task, so not on by default.